client/src/components/Notices/NoticeList.js

```diff
@@ -66,7 +66,9 @@ const NoticeList = ({
   return (
     <Container fluid className="notice-list">
       <Row>
-        <NoticeSidebar />
+        <Col xl="3" tag="aside" className="notice-list-sidebar">
```

Comment:
I wanted to move the className too, but it created UI bugs.

Reply:
Well I guess it is not a tragedy if it stays like this, but it would be fine to move it there.